pkg/kclient/pods.go

```diff
@@ -0,0 +1,118 @@
+package kclient
+
+import (
+	"io"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/pkg/errors"
+
+	// api resource types
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+const (
+	// waitForPodTimeOut controls how long we should wait for a pod before giving up
+	waitForPodTimeOut = 240 * time.Second
+)
+
+// WaitAndGetPod block and waits until pod matching selector is in in Running state
```

Comment:
Looks like the comment for this function is outdated because it doesn't just match `Running` phase since desiredPhase was added? Also, `in in`.

Reply:
Updated the comment

Reply:
Updated the comment to reflect the current status of the function.